doc/invites.md

```diff
@@ -0,0 +1,21 @@
+Guidelines for Instruqt invites
+===============================
+
+* Public title: `Replicated Hands-On Labs for [Vendor Name]`
+* Public description: `An opportunity for the [Vendor Name] team to explore the features and capabilities of Replicated in a lab environment.`
+* Internal description: ok to leave empty
+* Select tracks: 
+    * Select the tracks you want the Vendor to have access to. 
+    * Do not include tracks that are not relevant. For example, if they will not use Helm, do not include any Helm related tracks. 
+    * Also, order is important!
+* Content Restrictions:
+    * How long is content available to users after they access the invite?: Select 1 month.
```

Comment:
I like a limit for creating urgency, but it also means we have to create a new one down the road as more teams get engaged. Thoughts on that second bit?

Reply:
That would make for an ideal conversation. If more teams get engaged we would like to understand which tracks are ideal for them. What training / hands-on experience are the looking for. And that might result in a whole different invite, or extend the existing invite.